vsintegration/src/fsharp.vsreferences.targets

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@latkin  - I don't think there is a shared imported targets file for "vsintegration" - only for the compiler/library - so I think adding this file is ok and the best way to factor the swicth between VS references.